documentation/Database-Schema-Review.md

```diff
@@ -0,0 +1,99 @@
+# Database Schema Review
+
+![Database Schema](images/database-schema-review.png)
+
+## Database Area description: Review
+
+### Definition
+
+The review is part of the process of the application in the workflow to be approved or rejected in the system. For example a Drug Registration application, after being submited, goes through stages. In each one of these stages at least one review is required. To have an application reviewed, the system counts with special users called Reviewers (or evaluators).
+
+The Reviewer may or may not have permission to submit the LOQ (List of Questions) back to the Applicant that will be allowed to edit this selected LOQ.
+
+Actions will define which permission is required to submit the reviewed application. Which means that for some applications on a stage it can be required **more than one** review to done for the same section.
+
+First done by Reviewers with lowel `review level` permission, to add comments and decisions. This reviews will sit there until another review, called **consolidation**, is done (on top of the first ones). And the it will be submitted back to the Applicant. In some cases even one more review can be done on top of the second one - more details about that coming soon.
+
+## Tables
```

Comment:
I think this section needs an overall high-level explanation of the purpose of breaking it down the way we have. It could be a sentence or two at the top of each table section (like you've done for `review`, but for review_section_join, etc.) Or maybe a paragraph explaining the whole "review" tables structure here. Just listing the fields and what they refer to is not that helpful without a more conceptual justification for what the table is for overall.

Not sure the best way to achieve this, just putting the idea out there.

Reply:
Yes. I totally agree. I think I haven't done because I was also trying to see the whole picture. I'll try to add this now.